EveripediaNetwork/ep-ui
# Reworking filter tags

# Rework Filter Tags

This PR works on fixing issues with tag filtering

## How should this be tested?

1. Go to the event page
2. Click on any of the events filter
3. Check the change in layout 

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/3007
